src/components/flat-table/flat-table.stories.mdx

```diff
@@ -359,6 +357,17 @@ The `FlatTableRow` relies on specific child composition to identify sticky colum
 
 <ArgsTable of={Sort} />
 
+<TranslationKeysTable
+  translationData={[
+    {
+      name: "sort.accessibleName",
+      description: "The accessible name which is announced on interaction with the component.",
+      type: "func",
+      returnType: "string",
+    }
+  ]}
+/>
```

Comment:
**nitpick**: would you be happy to relocate the translation key table below the props section? This is to prevent it from being accidentially overlooked while browsing the page.

```mdx
# FlatTable

## Contents

## Quick Start

## Examples

## Props

## Translation keys

The following keys are available to override the translations for this component by passing in a custom locale object to the
[i18nProvider](https://carbon.sage.com/?path=/story/documentation-i18n--page).

<TranslationKeysTable translationData={[ /* ... */>]}>
```